garden_ai/local_data.py

```diff
@@ -1,23 +1,27 @@
 import json
 import logging
+from enum import Enum
 from typing import Dict, Union, Optional
 from pathlib import Path
 from uuid import UUID
 
 from garden_ai.gardens import Garden
 from garden_ai.pipelines import Pipeline
-from garden_ai.utils.misc import JSON
 
 LOCAL_STORAGE = Path("~/.garden").expanduser()
 LOCAL_STORAGE.mkdir(parents=True, exist_ok=True)
 
 logger = logging.getLogger()
 
 
+class ResourceType(Enum):
```

Comment:
This file has been rewritten to the point that I'd recommend looking at just the plain file instead of the diff: https://github.com/Garden-AI/garden/blob/will/search-part-two/garden_ai/local_data.py